### DISCUSSION TITLE
What will be the most efficient way to isolate operation (in our case 'cas') using memcached-proxy routelib.lua

### DISCUSSION THREAD

**Discussion by HMartyrossian** [memcached/memcached-proxylibs#12] [Q&A] [answered]
What will be the most efficient way to isolate operation ('cas', in our case) using Lua configuration file for the memcached-proxy routelib.lua?
Thank you!

**Comment by dormando** [comment, marked as answer]
@HMartyrossian can you explain what you want to do once it's isolated?

General documentation for the proxy is here: https://docs.memcached/org/features/proxy/

An example of mapping a single command to a specific route handler is here: https://github.com/memcached/memcached-proxylibs/blob/main/lib/routelib/examples/commandmap.lua - this is easy and efficient, but what would you like to do with it from there?

**Reply by HMartyrossian** [reply]
Hi @dormando :
Thank you very much for the memcached-proxy and your prompt response!
commandmap.lua - is an excellent example of what we would like to achieve.
Thank you!
